src/mantarray_file_manager/exceptions.py

```diff
@@ -56,3 +57,26 @@ def __init__(self, file_version: str):
         super().__init__(
             f"There is no supported migration path from version {file_version}. Supported paths are: {FILE_MIGRATION_PATHS}."
         )
+
+
+class UnsupportedArgumentError(Exception):
+    """Error raised if the arguments indicating the amount of centimilliseconds to be trimmed from the start and end are both None."""
```

Comment:
If all your error does is display a string, then generally you can just put the information about the error in the error message itself rather than creating extra things in a docstring.  Might as well display the information to the user as clearly as possible